core/test/financial-templates/Liquidatable.js

```diff
@@ -911,4 +912,39 @@ contract("Liquidatable", function(accounts) {
       assert.equal((await collateralToken.balanceOf(sponsor)).toString(), expectedPaymentSponsor.toString());
     });
   });
+  describe("Emergency shutdown", () => {
+    it("Liquidations are disabled if emergency shutdown", async () => {
+      // To mock the emergency shutdown, register a controlled EOA as the `Governor` within the `Finder`.
+      await finder.changeImplementationAddress(web3.utils.utf8ToHex("FinancialContractsAdmin"), mockGovernor, {
+        from: contractDeployer
+      });
+
+      // Create position.
+      await liquidationContract.create(
+        { rawValue: amountOfCollateral.toString() },
+        { rawValue: amountOfSynthetic.toString() },
+        { from: sponsor }
+      );
+      // Transfer synthetic tokens to a liquidator.
+      await syntheticToken.transfer(liquidator, amountOfSynthetic, { from: sponsor });
+
+      // Advance time until some point during contract life.
+      const expirationTime = await liquidationContract.expirationTimestamp();
+      await liquidationContract.setCurrentTime(expirationTime.toNumber() - 1000);
+
+      // Emergency shutdown the priceless position manager via liquidatable.
+      liquidationContract.emergencyShutdown({ from: mockGovernor });
```

Comment:
Why not go through the deployed FinancialContractsAdmin? E.g., call:

```
const fca = await FinancialContractsAdmin.deployed();
fca.callEmergencyShutdown(liquidationContract.address);
```

Is it because the permissions aren't set up correctly?

Reply:
I did it like this because this enables me to spoof calls from the Governor. I'll take a look if it's possible to do it this way; it should work fine as long as the address that is calling the `callEmergencyShutdown` has the correct permissions which the deployer address should (if i'm not mistaken) have be default. this will actually be a better solution.

Reply:
I have refactored the PR to use your method :)